src/System.Private.CoreLib/src/System/Reflection/RtFieldInfo.cs

```diff
@@ -188,6 +188,7 @@ internal override RuntimeModule GetRuntimeModule()
 
         public override object GetRawConstantValue() { throw new InvalidOperationException(); }
 
+#pragma warning disable CS8609 // TODO-NULLABLE: Covariant return types (https://github.com/dotnet/roslyn/issues/23268)
```

Comment:
The base class annotation should allow nullable instead.

Reply:
Fixed.